api/python/tests/test_supported_versions.py

```diff
@@ -15,12 +15,25 @@
 
 
 @pytest.fixture
-def initialized_CXGSchemaInfo(mock_load_supported_versions):
-    mock_load_supported_versions.return_value = {
+def ontology_info_content():
+    return {
         "5.0.0": {
-            "ontologies": {"CL": {"version": "v2024-01-01", "source": "http://example.com", "filename": "cl.owl"}}
+            "ontologies": {
+                "CL": {"version": "v2024-01-01", "source": "http://example.com", "filename": "cl.owl"},
+                "HANCESTRO": {
+                    "version": "v2024-01-01",
+                    "source": "http://example.com",
+                    "filename": "hancestro.owl",
+                    "additional_ontologies": ["FOO", "OOF"],
```

Comment:
What if two ontologies had the same `additional_ontologies`? It's valid in the schema.

```
"CL": {
   "additional_ontologies": ["FOO", "OOF"]
}
```

Reply:
Hm. I'll ask Brian if we want to account for this ever. It'd be easier to not allow that, otherwise we'd have to refactor everything in OntologyParser to optionally accept a param for which ontology you want to query from

Reply:
Per Brian's response in slack, I'm going to make a change to disallow repeat additional ontologies.